src/workers/postAddedSlackChannelSend.ts

```diff
@@ -38,6 +39,27 @@ export const postAddedSlackChannelSendWorker: TypedWorker<'api.v1.post-visible'>
         ]);
         const source = await post.source;
 
+        let adminJoinToken: string | undefined;
+
+        if (source.type === SourceType.Squad) {
```

Comment:
Isn't it only needed for private squads?

Reply:
Yeah good catch, for now actually better to specify the private check as well. I'll do that.

Reply:
Solved in bd11f05533a7bd445c602bd8c69e12344ea8d816